vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt

```diff
@@ -1020,9 +1022,6 @@ class VectorPreferences @Inject constructor(private val context: Context) {
         return defaultPrefs.getBoolean(SETTINGS_LABS_RENDER_LOCATIONS_IN_TIMELINE, true)
     }
 
-    /**
-     * Indicates whether or not thread messages are enabled
-     */
```

Comment:
Why this comment has vanished?

Reply:
It was an old comments. 
I've added a new comment for the setting key.

Reply:
I do not understand. Never mind.